tests/integration/conftest.py

```diff
@@ -739,3 +739,10 @@ def run_dataset_export_v2_task(cls,
 @pytest.fixture
 def export_v2_test_helpers() -> Type[ExportV2Helpers]:
     return ExportV2Helpers()
+
+
+@pytest.fixture
+def is_adv_enabled(client) -> bool:
```

Comment:
can we make it session scope? PS Not sure whether it is possible, but would save on extra http traffic

Reply:
good idea, added!